eng/InstallRuntimes.proj

```diff
@@ -16,19 +16,21 @@
     $(RepoRoot) - the root of the diagnostics repo
     $(RepositoryEngineeringDir) - the "eng" directory
     $(VersionsPropsPath) - path of Versions.props
+
+    From Directory.Build.props:
+    $(ArtifactsRuntimeDir) - artifacts\runtime directory
```

Comment:
The new directory name of 'runtime' feels a bit ambiguous to me. "dotnet-test" conveyed both that the files were in the dotnet layout (as opposed to CORE_ROOT layout) and also that the files were a test asset.